htdocs/api/v0.0.2/Candidates.php

```diff
@@ -218,7 +218,7 @@ protected function verifyField($data, $field, $values)
      * @param string $gender   Gender of the candidate to be created
      * @param string $PSCID    PSCID of the candidate to be created
      *
-     * @return none
+     * @return \Candidate
```

Comment:
I think these API changes are going to cause conflicts with #4244 and I'd prefer that one goes in first.

Maybe it's best to just temporarily add "htdocs/api" to the "exclude_analysis_directory_list" in the phan config since it's going to disappear as soon as @xlecours is done converting it to a module anyways.

Reply:
That's a good idea, will do

Reply:
@driusan I'll make a separate PR to add `htdocs/` to the exclude_analysis_directory_list because I think this is likely to come up in the other Phan PRs. It'll be easier to merge that one first and rebase the others.

Reply:
'htdocs' shouldn't be excluded, only 'htdocs/api'. Other php files in the 'htdocs' directory should still be analyzed.